internal/command/action_test.go

```diff
@@ -240,3 +240,55 @@ testprovider-trailingnewline version 3.2.1
 		So(output, ShouldEqual, expected)
 	})
 }
+
+func TestLocateFileRecurseUp(t *testing.T) {
+	filename := "test.txt"
+	currentDir, _ := os.Getwd()
+	var err error
```

Comment:
Pre-declaring the error is unnecessary - just change the `os.Create`s to have `:=` instead of `=`